pytestqt/_tests/test_wait_signal.py

```diff
@@ -283,3 +283,21 @@ def test_wait_twice(qtbot, timer, multiple, do_timeout, signaller):
             signaller.signal.emit()
         with func(arg):
             signaller.signal.emit()
+
+
+@pytest.mark.skipif(QT_API == 'pyside', reason='test crashes PySide')
+def test_destroyed(qtbot):
+    """Test that waitSignal works with the destroyed signal (#82).
+
+    For some reason, this crashes PySide although it seems perfectly fine code.
```

Comment:
Sorry - I was trying to say "maybe it segfaulted with PySide because `sip.isdeleted` raised an exception and that caused a segfault for some reason" - but I guess it already segfaulted *before* you added `sip.isdeleted` :wink: